packages/app/pages/dev.tsx

```diff
@@ -43,21 +44,54 @@ export default function DevPanel(): ReactElement {
         <Button
           style={{ marginRight: '15px' }}
           type="default"
-          onClick={() => API.seeds.delete()}
+          onClick={() => {
+            API.seeds
```

Comment:
Consider moving it to the method

Reply:
erm, move it to what method? like moving the `.then` and `.catch` into the API's `delete()` method?